kubernetes-kubelet/config.json.template

```diff
@@ -370,7 +370,7 @@
           }
     ],
     "linux": {
-        "rootfsPropagation": "rslave",
+        "rootfsPropagation": "rshared",
```

Comment:
wouldn't the /var/lib/kubelet rshared be enough?  If it is not sufficient, as it is not a mount point on the host, we will need to ensure it is.

This is what we do, for example, with the CRI-O system container:

https://github.com/projectatomic/atomic-system-containers/blob/master/cri-o-centos/set_mounts.sh
https://github.com/projectatomic/atomic-system-containers/blob/master/cri-o-centos/Dockerfile#L26
https://github.com/projectatomic/atomic-system-containers/blob/master/cri-o-centos/service.template#L7

The risk of having rootfsPropagation=rshared is that mount events can get out of control, and you will end up with a lot of undesired mounts in the host mount namespace.

Reply:
I think you're right, i'll rerun some tests and let you know.